test/unit/sth_test.js

```diff
@@ -969,6 +969,110 @@ describe('Data removal', function() {
   );
 });
 
+describe('POST /admin/log', function() {
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-sth-comet/commit/115dbc8419c8e100b8ea0e8bb045a0a810a7f131